app-backend/db/src/main/scala/com/azavea/rf/database/LayerAttributeDao.scala

```diff
@@ -96,6 +95,13 @@ object LayerAttributeDao extends Dao[LayerAttribute] {
       .to[List]
   }
 
+  def unsafeMaxZoomForLayer(layerName: String): ConnectionIO[(String, Int)] = {
```

Comment:
@jisantuc A heads up that I rebased this branch (all of the conflicts were in this file) to get rid of merge conflicts so that CI could run on dependent PRs.